packages/block-library/src/navigation/style.scss

```diff
@@ -522,6 +521,16 @@
 	}
 }
 
+.wp-block-navigation__plain-menu {
+	display: none;
+
+	&:not(.always-shown) {
+		@include break-small {
+			display: block;
```

Comment:
If we want the flex layout settings to reflect correctly in the nav contents, this class has to duplicate the responsive container content styles [here](https://github.com/WordPress/gutenberg/blob/trunk/packages/block-library/src/navigation/style.scss#L384).